bigchaindb/util.py

```diff
@@ -1,4 +1,12 @@
+
```

Comment:
general thought: what goes in util?
- helper functions
- transaction functions
- singletons/factories
- ...

Currently it looks a bit like a container where we store stuff that doesnt fit into core (node logic). We'll probably need to reshuffle this later